api/src/services/message-service.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var MessageService = {};
+MessageService.name = 'MessageService';
+
+var { ForbiddenWord } = require('../models');
+
+MessageService.cleanMessage = async(workspaceId, message) => {
+  try {
+    var forbiddenWords = await ForbiddenWord.findAll({
+      where: { workspaceId },
+    });
+    var words = forbiddenWords.map(function(w){
```

Comment:
acá estás creando un array que contiene objetos que cada uno tiene una sola key, es ridiculo. Encima más abajo estás haciendo un doble for para recorrer esta estructura extraña que armaste.

Por qué no hacer directamente:

```
forbiddenwords.forEach((forbiddenword) => {
        var regEx = new RegExp(forbiddenword.word, 'ig');
        message = message.replace(regEx, forbiddenword.replaceby);
})
```